### ISSUE TITLE
importing TimeSeries fails with astropy version 4.3.post1

### ISSUE THREAD

**Issue by esanger** [gwpy/gwpy#1375]
Importing TimeSeries fails with when using astropy version 4.3.post1. With astropy version 4.2.1 it is working fine.
```python
from gwpy.timeseries import TimeSeries
```
```python
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
/tmp/ipykernel_34793/2783767739.py in <module>
----> 1 from gwpy.timeseries import TimeSeries

~/.local/lib/python3.8/site-packages/gwpy/timeseries/__init__.py in <module>
     20 """
     21 
---> 22 from .core import (TimeSeriesBase, TimeSeriesBaseDict, TimeSeriesBaseList)
     23 from .timeseries import (TimeSeries, TimeSeriesDict, TimeSeriesList)
     24 from .statevector import (StateVector, StateVectorDict, StateVectorList,

~/.local/lib/python3.8/site-packages/gwpy/timeseries/core.py in <module>
     51 from gwosc.api import DEFAULT_URL as GWOSC_DEFAULT_HOST
     52 
---> 53 from ..types import Series
     54 from ..detector import (Channel, ChannelList)
     55 from ..segments import SegmentList

~/.local/lib/python3.8/site-packages/gwpy/types/__init__.py in <module>
     24 
     25 
---> 26 from .array import Array
     27 from .array2d import Array2D
     28 from .index import Index

~/.local/lib/python3.8/site-packages/gwpy/types/array.py in <module>
     36 from astropy.units import Quantity
     37 
---> 38 from ..detector import Channel
     39 from ..detector.units import parse_unit
     40 from ..time import (Time, to_gps)

~/.local/lib/python3.8/site-packages/gwpy/detector/__init__.py in <module>
     25 import datetime
     26 
---> 27 from .channel import (Channel, ChannelList)
     28 from . import (units, io)
     29 

~/.local/lib/python3.8/site-packages/gwpy/detector/channel.py in <module>
     31 from ..time import to_gps
     32 from ..utils.misc import if_not_none
---> 33 from .units import parse_unit
     34 
     35 __author__ = 'Duncan Macleod <duncan.macleod@ligo.org>'

~/.local/lib/python3.8/site-packages/gwpy/detector/units.py in <module>
    146 
    147 # enable imperial units
--> 148 units.add_enabled_units(units.imperial)
    149 
    150 # -- custom units settings

AttributeError: module 'astropy.units' has no attribute 'imperial'
```

**Comment by duncanmmacleod** [comment]
This was fixed in #1345, so will be corrected in the next release.

**Comment by danilosaft** [comment]
Meanwhile, as per https://keflavich-astropy.readthedocs.io/en/latest/units/ you can do

```
from astropy.units import imperial
imperial.enable() 

```
before the gwpy import
